Update MTA integration docs: add bus tracking support (#43582)
home-assistant/home-assistant.github.io: source/_integrations/mta.markdown

ha_domain: mta
ha_platforms:
  - sensor
ha_integration_type: service
ha_quality_scale: silver
---

The **MTA New York City Transit** {% term integration %} provides real-time subway and bus arrival predictions for NYC transit lines using GTFS-RT data from the [Metropolitan Transportation Authority (MTA)](https://new.mta.info/).

## Prerequisites

- **Subway tracking**: No API key is required.
- **Bus tracking**: An [MTA Bus Time API key](https://bustime.mta.info/wiki/Developers/Index) is required.

{% include integrations/config_flow.md %}

{% configuration_basic %}
API key:

>   description: "Your MTA Bus Time API key. Required for bus tracking, optional for subway only."

Author: There are multiple sentences.

{% endconfiguration_basic %}

## Adding a subway stop

To add a subway arrival sensor:

1. Go to {% my integrations title="**Settings** > **Devices & services**" %} and select the **MTA New York City Transit** integration.
2. Select **Add subway stop**.
3. Select the subway line you want to monitor from the dropdown.
4. Select the stop and direction (indicated by N/S suffix for northbound/southbound).

## Adding a bus stop

To add a bus arrival sensor:

1. Go to {% my integrations title="**Settings** > **Devices & services**" %} and select the **MTA New York City Transit** integration.
2. Select **Add bus stop**.
3. Enter the bus route (for example, `M15`, `B46`, `Q10`).
4. Select the stop from the list. Direction information is included with each stop.

{% note %}

> Bus tracking requires an API key. If you did not provide one during initial setup, you will be prompted to add one through the reauthentication flow.

Reviewer: ```suggestion
Bus tracking requires an API key. If you did not provide one during initial setup, you will be prompted to enter one when you attempt to add a bus stop.
```
